### DISCUSSION TITLE
Wacom  One graphics tablet driver in OrangePi5Plus

### DISCUSSION THREAD

**Discussion by sarbo** [Joshua-Riek/ubuntu-rockchip#1063] [Ideas]
It would be nice to have the driver for the Wacom graphics tablets  for OrangePi5Plus.

**Comment by Joshua-Riek** [comment]
It would be nice for you to send a pull request to enable this driver.

**Comment by sarbo** [comment]
I did not intend  to sound  sarcastic...  I will  add it to my list of things to do.  I  use  the  graphics tablet for Physics notes ....   example...  
[wave_equation.pdf](https://github.com/user-attachments/files/17172360/wave_equation.pdf)

**Comment by sarbo** [comment]
See  https://github.com/linuxwacom/input-wacom/wiki/Installing-input-wacom-from-source

sea@sea-desktop:~$ modinfo wacom 
filename:       /lib/modules/6.1.0-1025-rockchip/extra/wacom.ko
license:        GPL
description:    USB Wacom tablet driver
author:         Vojtech Pavlik <vojtech@ucw.cz>
version:        v2.00-input-wacom-master
srcversion:     D4DB3C43618D8C68DA94975

**Comment by sarbo** [comment]
So many  problems  with this   hardware  and software... it is   not usable ...   where do I begin...    not much  software  runs  on this.......  I have  had to  reinstall  twice because   of  conflicts....... ok  for  play....  not ready  for   prime  time!    ......
